### DISCUSSION TITLE
Student Developer Pack - Domains

### DISCUSSION THREAD

**Discussion by jack-sisti** [community/community#69808] [GitHub Education] [answered]
### Hi everyone,

Hello!

I have access to the student dev. pack, and saw that we can get free domains as a student. I also noticed that some of the companies offering these domains are offering more than one TLD, and I am now wondering if those domain hosting services give us the option to choose _one_ of those domains (per company, of course), or if we are able to get one of each of those TLD's.

I apologize if this question does not make sense.

Thanks!

Jack Sisti

### A suggestion...(optional)

more specification on particular aspects of the student developer pack

**Comment by mantasu** [comment]
One of each, i.e., you get access to everything that is listed in the student dev pack.

**Reply by jack-sisti** [reply]
i'm having a hard time describing what I mean...

there are three "tools" listed in the dev. pack, under the "domains" category.
one of these tools, Name.com, has the following description:

"Build your online presence with a free .LIVE domain name, free privacy protection, and a free SSL certificate."

When I search "name.com student domain" (on google), and click on (https://www.name.com/partner/github-students) the first link it shows a bunch of TLDs. 

I am wondering if name.com is offering only one domain, and I must choose which one of those I want
or
if name.com is offering one domain for each listed TLD

thanks again!

**Reply by mantasu** [reply, marked as answer]
Oh sorry, I checked `Name.com` and it seems to only offer 1 free TLD as stated [here](https://www.name.com/support/articles/360036364834-signing-up-for-github-student-developer-pack).

**Reply by jack-sisti** [reply]
I see.
Thank you so much for your help!

**Reply by mantasu** [reply]
No worries, enjoy your benefits :)
